src/modules/player.js

```diff
@@ -120,6 +121,26 @@ class Player {
                         this.winCondition = true;
                     }
                 }
+				//handle slow wall collision
+				else if (unsortedEntities[i].name == "slowWall") {
+                    let colliding_on_x_axis = (newX + this.size > unsortedEntities[i].x) && (newX < unsortedEntities[i].x + unsortedEntities[i].size);
+                    let colliding_on_y_axis = (newY + this.size > unsortedEntities[i].y) && (newY < unsortedEntities[i].y + unsortedEntities[i].size);
+                    let colliding = colliding_on_x_axis && colliding_on_y_axis;
+					
+					if (colliding) {
+                        this.speed = 0.8;
+                    }
+				}
+				//handle reverse slow wall collision
+				else if (unsortedEntities[i].name == "antiSlow") {
```

Comment:
Similar here, do you want it to be called `antiSlow` or `reverseSlow`? Might be helpful for future code readers if the class name, the entity name, and file name all match if possible :)